index.html

```diff
@@ -46,8 +46,169 @@ <h2>I'm a Software Developer</h2>
+                <h2>Profesional Art <br>
+                    Printing Data
+                </h2>
+                <p>
+                    A daily selection of privately  <br> 
+                    personalized reads; no accounts or <br> 
+                    sign-ups required. Has been the <br> 
+                    industry's standard.
+                </p>
+                <div class="btn">
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="see-btn">
+                    <button>See Projects</button>
+                </div>
+            </div>
+            <div class="project-section">
+                <h2>Profesional Art <br>
+                    Printing Data
+                </h2>
+                <p>
+                    A daily selection of privately  <br> 
+                    personalized reads; no accounts or <br> 
+                    sign-ups required. Has been the <br> 
+                    industry's standard.
+                </p>
+                <div class="btn">
+                    <ul>
+                        <li>HTML</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="see-btn">
+                    <button>See Projects</button>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="about-me-section">
+        <div class="about-me">
+            <h1>About me</h3>
+            <p>
+                Hello I’m a software developer! I can help <br>
+                you build a product , feature or website <br>
+                Look through some of my work and <br> 
+                experience! If you like what you see and 
+                <br> have a project you need coded, don’t 
+                <br> hesitate to contact me.
+            </p>
+            <div class="resume-btn">
+                <button>Get my resume</button>
+            </div>
+        </div>
+        <hr>
+        <div class="about_footer">
+            <div class="profile">
+               <ul>
+                   <li class="list-style Language">Language
+                       <ul>
+                            <li>JavaScript</li>
+                            <li>Ruby</li>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
+                    </li>
+                    <li class="list-style framework">Framework
+                        <ul>
+                            <li>React.js</li>
+                            <li>Ruby on Rails</li>
+                            <li>RSpec</li>
+                            <li>Capybara</li>
+                         </ul>
+                     </li>
+                     <li class="list-style skills">Skills
```

Comment:
- [ ] I recommend making the Skills title centered. As this will make your project match the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=1-1471&t=bZX5VS91nNO9rtwR-0).

## Your design:
![image](https://user-images.githubusercontent.com/86069740/236591607-2e0c00bf-56e6-4392-93ec-2e1db8dad951.png)

## Figma
![image](https://user-images.githubusercontent.com/86069740/236591621-4bb39f06-b2fc-48a3-ba11-b5b7f998ab9b.png)